### ISSUE TITLE
Ubuntu Linux: Exception calling "AddTestResult" with "8" argument(s)

### ISSUE THREAD

**Issue by nvarscar** [pester/Pester#1182]
T## 1. General summary of the issue
**It** block throws an error after successful completion on Linux Ubuntu environment in Appveyor:
```
Exception calling "AddTestResult" with "8" argument(s): "The attribute cannot be added because variable Result with value  would no longer be valid."
```
It runs perfectly fine on Windows.

## 2. Describe Your Environment
Operating System, Pester version, and PowerShell version:
Ubuntu 18.04 LTS
PowerShell Core 6.1.1
Latest Pester installed from PSGallery

The error occurs during Appveyor CI build, that's the best I can get.

Erroneous build:
https://ci.appveyor.com/project/nvarscar/dbops/build/job/qavfvq159x6tmt1e
Errors exported via `Export-CliXml`
https://ci.appveyor.com/api/buildjobs/qavfvq159x6tmt1e/artifacts/dbops_log_errors.xml
`Import-Clixml -Path 'E:\tmp\dbops_log_errors.xml' | ? Exception -like '*AddTestResult*'|select -first 1 *`

## 3. Expected Behavior

No errors :)

## 4.Current Behavior

Tests fail miserably

## 6. Context

Trying to run tests in both Windows and Linux to confirm full Xplat compatibility.

**Comment by nohwnd** [comment]
Thanks for reporting this seems to affect only Ubuntu on AppVeyor, all my builds in docker and on Travis run just fine. We are already trying to figure out the error in #1101

**Comment by nvarscar** [comment]
Okay, got it. I'm digging through the logs now, there are actually proper test results in the `$error` array after all. Tests are indeed failing, but the error thrown is not the one that is expected.
Thanks for looking into it.

**Comment by nohwnd** [comment]
This is due to bug PowerShell 6.1 that makes Hashtables case sensitive on Linux, see #1101, and fixed by PR and released as 4.4.4 .

**Comment by nohwnd** [comment]
@nvarscar Chris @bergmeister and the others that participated are the ones you should thank. I just published it :)
